### PR TITLE
Hopefully another fun, but valuable tip to close 2021.

### PR DESCRIPTION
**Pull request by wpschaub**

### CODE REVIEW

**Review by wsbctechnicalblog — COMMENTED** [review]
Great addition to your previous email article. There is a typo on line 19: "hours of previous time"; I believe you wanted to write 'precious time'

**Review by erwsch1 — DISMISSED** [review]

**Comment by wsbctechnicalblog** [comment]
@wpschaub are you good with the spelling correction I suggested, or would you like me to wait for you before I merge & publish?

**Comment by wpschaub** [comment]
@wsbctechnicalblog I have fixed the typo. Thanks for your Eagle vision! good to launch.

**Review by wsbctechnicalblog — APPROVED** [review]

**Pull request merged**
